### PR TITLE
companion for substrate/pull/6132

### PR DESCRIPTION
**Pull request by kianenigma**
Also did a bit of cleanup in all runtime files, see here: https://github.com/paritytech/substrate/pull/6132#issuecomment-634509086

### CODE REVIEW

File: runtime/westend/src/lib.rs [hunk 1]
```diff
@@ -61,12 +62,7 @@ use session::historical as session_historical;
 
 #[cfg(feature = "std")]
 pub use staking::StakerStatus;
-#[cfg(any(feature = "std", test))]
-pub use sp_runtime::BuildStorage;
-pub use timestamp::Call as TimestampCall;
-pub use balances::Call as BalancesCall;
```
Comment (bkchr) [inline runtime/westend/src/lib.rs:67]: Why do you remove them?

File: runtime/westend/src/lib.rs [hunk 2]
```diff
@@ -179,10 +175,16 @@ impl balances::Trait for Runtime {
 
 parameter_types! {
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
-	// for a sane configuration, this should always be less than `AvailableBlockRatio`.
 	pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
 }
 
+// for a sane configuration, this should always be less than `AvailableBlockRatio`.
+const_assert!(
```
Comment (bkchr) [inline runtime/westend/src/lib.rs:182]: Are we 100% sure that the compiler does not include anything from that into the final binary?

File: runtime/polkadot/src/lib.rs [hunk 3]
```diff
@@ -67,12 +67,7 @@ use static_assertions::const_assert;
 
 #[cfg(feature = "std")]
 pub use staking::StakerStatus;
-#[cfg(any(feature = "std", test))]
-pub use sp_runtime::BuildStorage;
-pub use timestamp::Call as TimestampCall;
-pub use balances::Call as BalancesCall;
```
Comment (bkchr) [inline runtime/polkadot/src/lib.rs:73]: Same.

File: runtime/common/src/lib.rs [hunk 4]
```diff
@@ -38,33 +38,27 @@ use frame_support::{
...
 	pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
-	pub const MaximumExtrinsicWeight: Weight = AvailableBlockRatio::get()
-		.saturating_sub(Perbill::from_percent(10)) * MaximumBlockWeight::get();
+	pub MaximumExtrinsicWeight: Weight = AvailableBlockRatio::get()
+		.saturating_sub(AVERAGE_ON_INITIALIZE_WEIGHT) * MaximumBlockWeight::get();
 	pub const MaximumBlockLength: u32 = 5 * 1024 * 1024;
 }
 
+const_assert!(AvailableBlockRatio::get().deconstruct() >= AVERAGE_ON_INITIALIZE_WEIGHT.deconstruct());
+
```
Comment (bkchr) [inline runtime/common/src/lib.rs:61]:
```suggestion
```

File: runtime/westend/src/lib.rs (hunk 1 above)
Comment (kianenigma) [reply runtime/westend/src/lib.rs:67]: > Also did a bit of cleanup in all runtime files, see here: https://github.com/paritytech/substrate/pull/6132#issuecomment-634509086

File: runtime/westend/src/lib.rs (hunk 2 above)
Comment (kianenigma) [reply runtime/westend/src/lib.rs:182]: Not by experiment, but I have asked ppl before (including you probably) and the answer was yes, we are sure. 

it was already in all of the runtimes except one of them anyhow though.

Comment (kianenigma) [reply runtime/westend/src/lib.rs:182]: Not by experiment, but I have asked ppl before (including you probably) and the answer was yes, we are sure. 

it was already in all of the runtimes except one of them anyhow though.

File: runtime/westend/src/lib.rs (hunk 1 above)
Comment (bkchr) [reply runtime/westend/src/lib.rs:67]: Yeah that is for example used by Cumulus :D

File: runtime/westend/src/lib.rs [hunk 5]
```diff
@@ -66,7 +67,7 @@ pub use sp_runtime::BuildStorage;
 pub use timestamp::Call as TimestampCall;
 pub use balances::Call as BalancesCall;
 pub use attestations::{Call as AttestationsCall, MORE_ATTESTATIONS_IDENTIFIER};
-pub use parachains::Call as ParachainsCall;
+use parachains::Call as ParachainsCall;
```
Comment (kianenigma) [inline runtime/westend/src/lib.rs:70]:
```suggestion
pub use parachains::Call as ParachainsCall;
```

File: runtime/polkadot/src/lib.rs [hunk 6]
```diff
@@ -72,7 +72,7 @@ pub use sp_runtime::BuildStorage;
 pub use timestamp::Call as TimestampCall;
 pub use balances::Call as BalancesCall;
 pub use attestations::{Call as AttestationsCall, MORE_ATTESTATIONS_IDENTIFIER};
-pub use parachains::Call as ParachainsCall;
+use parachains::Call as ParachainsCall;
```
Comment (kianenigma) [inline runtime/polkadot/src/lib.rs:75]:
```suggestion
pub use parachains::Call as ParachainsCall;
```

**Comment by kianenigma** [comment]
reverted.

**Pull request merged**